paperweight-lib/src/main/kotlin/util/McDev.kt

```diff
@@ -41,7 +41,7 @@ object McDev {
         patches: Iterable<Path>,
         decompJar: Path,
         importsFile: Path?,
-        librariesDir: Path?,
+        librariesDirs: List<Path>?,
```

Comment:
move this argument down one line, make it non null, and default value of `listOf()`
```kotlin
librariesDirs: List<Path> = listOf(),
```

callers will need updating ofc